src/cli/extensions/helpers.js

```diff
@@ -9,7 +9,10 @@ import { merge } from '../../configuration';
 import { warningLabel, feedbackMessage } from '../../helpers/style';
 import ExtensionError from './error';
 
+const debug = require('debug')('roc:core:cli:extensions');
```

Comment:
Can have it either way. I like declaring debug in a one-liner. It is simple and explicit. What is gained from using the module syntax? In my eyes it just adds some extra noise without gains in this case, as it is spread across multiple lines.